README.md

```diff
@@ -37,7 +37,7 @@ Make sure both mingw's and the sdk's `bin` directories are in your `PATH` e.g. (
 ```
 C:\> set PATH="C:\mingw-w64\bin;C:\gtk\bin;%PATH%"
 ```
-It's crucial that GCC from mingw is used by Rust so either make sure that mingw is earlier in the `PATH` or delete `gcc.exe` and `ld.exe` from the Rust installation.
+You will get a linking error `ld: cannot find -limm32` if `gcc.exe` and `ld.exe` that come with Rust are present in its `bin` directory so make sure they're not there.
```

Comment:
Hum... I didn't read it like that. I'd replace "that" by "which" in this case. But I'm not an english native (and maybe you are !), so ?